force-app/main/default/triggers/CheckContainerCapacity.trigger

```diff
@@ -0,0 +1,8 @@
+trigger CheckContainerCapacity on Car_Part__c (before insert, before update) {
```

Comment:
CarPart object has already created CarPartTrigger and CarPartTriggerTriggerHandler. Please check videos how our architecture set.

Reply:
Delete this trigger